### PR TITLE
CB-10845 Invalidate manifest cache in prepare

### PR DESCRIPTION
**Pull request by vladimir-kotikov**
This fixes [CB-10845](https://issues.apache.org/jira/browse/CB-10845)

### CODE REVIEW

File: template/cordova/lib/prepare.js [hunk 1]
```diff
@@ -420,6 +420,11 @@ module.exports.prepare = function (cordovaProject) {
     this._config = updateConfigFilesFrom(cordovaProject.projectConfig,
         this._munger, this.locations);
 
+    // CB-10845 avoid using cached appxmanifests since they could be
+    // previously modififed outside of AppxManifest class
+    // TODO: invalidate only entries that were affected by config munge
+    AppxManifest.purgeCache();
```
Comment (omefire) [inline template/cordova/lib/prepare.js]: In this instance, who modifies the file outside the 'AppxManifest' class ?

Comment (vladimir-kotikov) [inline template/cordova/lib/prepare.js]: It is a `_munger` which applies the changes defined by `<config-file>` elements from `plugin.xml`

**Comment by omefire** [comment]
LGTM!

**Comment by vladimir-kotikov** [comment]
Thanks, @omefire

File: template/cordova/lib/prepare.js (hunk 1 above)
Comment (omefire) [inline template/cordova/lib/prepare.js]: Thanks, it makes sense

**Pull request closed without merging**